scripts/index.js

```diff
@@ -283,6 +283,10 @@ function showModal (vRepoId, oEvent) {
           : "";
   sHTML = sHTML.replace("[[description]]", sDescription);
 
+  let sTopics = oRepo.topics ? oRepo.topics.map((t) => "<span class='pill'>"+t+"</span>").join(" ") : "";
+  sHTML = sHTML.replace("[[topics]]", sTopics);
+  sHTML = sHTML.replace(/\[\[#if topics\]\](.*)\[\[\/if\]\]/, sTopics ? "$1" : "");
```

Comment:
yes, that should work. It is a quick & dirty templating solution

Reply:
Yeah, I figured :)